src/hooks/usePopover.ts

```diff
@@ -3,7 +3,7 @@ import { usePopper } from "react-popper";
 import { Placement } from "./popoverConstants";
 
 import useForceUpdate from "./useForceUpdate";
-import * as PopperJS from "@popperjs/core";
+import { Options, State } from "@popperjs/core";
```

Comment:
```suggestion
import type { Options, State } from "@popperjs/core";
```

Should make sure it doesn't find its way into the bundle 🙂